modules/gapi/test/gapi_frame_tests.cpp

```diff
@@ -14,47 +14,21 @@
 namespace opencv_test {
 
 G_API_OP(GBlurFrame, <GMat(GFrame)>, "test.blur_frame") {
```

Comment:
Does it mean that we have to overwrite all operation for `core` and `imgproc` for `GFrame` ?

Reply:
No. Read the proposal